bin/game.rb

```diff
@@ -0,0 +1,103 @@
+require_relative '../lib/board.rb'
+
+class Game
+
+    def initialize
+        @board = Board.new()        
+        @moves = 0
+    end  
+
+    def show()
```

Comment:
As I stated above, in requirements to on how the project should work, should avoid puts in Class logic

I am thinking there should be another way to do this. how can you return this? maybe use of `\n` is a neasy fix

Or you can refactor code in a way that you use an array

Here's an example of how I did it

```ruby
        
        def initialize(grid_board = [])
	  @grid = grid_board.empty? ? default_board : grid_board
	end
        def default_board
		Array.new(3) { Array.new(3) { Cell.new } }
	end
        def formatted_grid
		grid.each_with_index do |row, outer|
			r = row.each_with_index.map do |col, inner|
				col.value.empty? ? inner +  1 + (row.length * outer) : col.value
			end
			puts r.join(' ')
		end
	end
```


And of course we had attr_reader for `:grid`